core/jvm/src/test/scala/me/shadaj/scalapy/py/ReferenceCountStressTest.scala

```diff
@@ -0,0 +1,24 @@
+package me.shadaj.scalapy.py
+
+import org.scalatest.funsuite.AnyFunSuite
+
+class ReferenceCountStressTest extends AnyFunSuite {
+  val gc = module("gc")
+  val sys = module("sys")
+  def referenceCount(obj: Dynamic): Int = sys.getrefcount(obj).as[Int]
+
+  test("Repeated global function calls maintain constant reference counts") {
+      val slice = global.slice(0)
+      val baseCount = referenceCount(slice)
+      println(baseCount)
+      (0 to 10000).foreach { _ =>
+        System.gc()
+        System.runFinalization()
+        gc.collect()
+
+        val count = referenceCount(slice)
+        // println(count)
```

Comment:
Could you remove these lines for now to get this merged in and use this test as just a stress test to catch segfaults? We can add in more precise reference count checks in a later PR perhaps?